controllers/IndexController.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+/**
+ * include the exhibitHelper.php
+ */
+require_once __DIR__ . '/../helpers/exhibitHelper.php';

```

Comment:
file naming conventions are to use PascalCase and name the file after the plugin, so this should be renamed HeadsUpFunctions.php and updated everywhere